config-freebsd.md

```diff
@@ -0,0 +1,124 @@
+
+Optionally, additional devices can be exposed to the container using an array of entries inside the `devices` root field:
+
+* **`path`** _(string, REQUIRED)_ - the device path relative to `/dev`
+* **`mode`** _(integer, OPTIONAL)_ - device permissions as an integer which is interpreted as in `chmod(1)`.
+
+### Example
+```json
+"devices": [
+	{
+        "path": "pf",
+        "mode": "0700"
+    }
+]
+```
+
+## <a name="configFreeBSDJail" />Jail
+
+On FreeBSD, containers are implemented using the platform's jail subsystem. Configuration for the container jail is via fields inside the `jail` root field:
+
+* **`parent`** _(string, OPTIONAL)_ - parent jail.
+    The value is the name of a jail which should be this container's parent (defaults to none). This can be used to share namespaces such as `vnet` with another container.
+* **`host`** _(string, OPTIONAL)_ - allow overriding hostname, domainname, hostuuid and hostid.
+    The value can be "new" which allows these values to be overridden in the container or "inherit" to use the host values (or parent container values). If set to "new", the values for hostname and domainname are taken from the base config, if present.
+* **`ip4`** _(string, OPTIONAL)_ - control the availability of IPv4 addresses.
+    This is typically left unset if the container has a vnet, set to "inherit" to allow access to host (or parent container) addresses or set to "disable" to stop use of IPv4 entirely.
+* **`ip6`** _(string, OPTIONAL)_ - control the availability of IPv6 addresses.
+    This is typically left unset if the container has a vnet, set to "inherit" to allow access to host (or parent container) addresses or set to "disable" to stop use of IPv6 entirely.
+* **`vnet`** _(string, OPTIONAL)_ - control the vnet used for this container.
+    The value can be "new" which causes a new vnet to be created for the container or "inherit" which shares the vnet for the parent container (or host if there is no parent).
+* **`sysvmsg`** _(string, OPTIONAL)_ - allow access to SYSV IPC message primitives.
+    If set to "inherit", all IPC objects on the system are visible to this container, whether they were created by the container itself, the base system, or other containers.  If set to "new", the container will have its own key namespace, and can only see the objects that it has created; the system (or parent container) has access to the container's objects, but not to its keys.  If set to "disable", the container cannot perform any sysvmsg-related system calls.
+* **`sysvsem`** _(string, OPTIONAL)_ - allow access to SYSV IPC semaphore primitives, in the same manner as sysvmsg.
+* **`sysvshm`** _(string, OPTIONAL)_ - allow access to SYSV IPC shared memory primitives, in the same manner as sysvmsg.
```

Comment:
These options are listed as optional but do not define a default.  I would suggest that 'new' is a good balance between security and usability as a default.